.gitignore

```diff
@@ -11,3 +11,5 @@ pytestdebug.log
 
 fixtures/
 /docs/_build
+
+.idea/
```

Comment:
your own IDE files shouldn't be added to the gitignore of every project, instead configure a global gitignore

https://help.github.com/articles/ignoring-files/

Reply:
hmm good idea, thank you